$NetBSD: patch-aa,v 1.3 2003/12/01 14:23:51 jdolecek Exp $

This is PHP CVS tree now, remove patch from pkgsrc on next PHP update.

--- config.m4.orig	2003-11-29 14:53:04.000000000 +0100
+++ config.m4	2003-11-29 14:53:13.000000000 +0100
@@ -7,7 +7,7 @@
 
 if test "$PHP_BCMATH" != "no"; then
   AC_DEFINE(WITH_BCMATH, 1, [Whether you have bcmath])
-  PHP_NEW_EXTENSION(bcmath, bcmath.c number.c \
+  PHP_NEW_EXTENSION(bcmath, bcmath.c \
 libbcmath/src/add.c libbcmath/src/div.c libbcmath/src/init.c libbcmath/src/neg.c libbcmath/src/outofmem.c libbcmath/src/raisemod.c libbcmath/src/rt.c libbcmath/src/sub.c \
 libbcmath/src/compare.c libbcmath/src/divmod.c libbcmath/src/int2num.c libbcmath/src/num2long.c libbcmath/src/output.c libbcmath/src/recmul.c \
 libbcmath/src/sqrt.c libbcmath/src/zero.c libbcmath/src/debug.c libbcmath/src/doaddsub.c libbcmath/src/nearzero.c libbcmath/src/num2str.c libbcmath/src/raise.c \
